retail-managment-system/rms-ui/src/app/core/layout/header/header.component.html

```diff
@@ -9,9 +9,12 @@
     >
 
     <div class="collapse navbar-collapse">
-      <button class="btn btn-outline-light ms-auto" type="submit">
+      <button class="btn btn-outline-light ms-auto" type="submit" (click)="login()">
         {{ "login" | translate }}
       </button>
+      <button class="btn btn-outline-light ms-3" type="submit" (click)="logout()">
+        Log out
+      </button>
```

Comment:
login or logout appear depend on user logged on or not

Reply:
done